# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpChannelParams.ipdlh
# Commit: 709f8d50aa33
# Full Hash: 709f8d50aa33f299e1e72366c59b1c28ad0b547b
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-24 09:31:07
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P10 wait PHttpChannel::OnStartRequestSent for permission/cookie update from parent, r=mayhemer,necko-reviewers
#   
#   Depends on D77751
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D77752
# ==============================================================================

diff -r 3b268ac50692 -r 709f8d50aa33 netwerk/protocol/http/HttpChannelParams.ipdlh
--- a/netwerk/protocol/http/HttpChannelParams.ipdlh	Tue Jun 23 16:58:33 2020 +0000
+++ b/netwerk/protocol/http/HttpChannelParams.ipdlh	Tue Jun 23 16:58:36 2020 +0000
@@ -48,6 +48,7 @@
   nsCString appCacheGroupId;
   nsCString appCacheClientId;
   nsIReferrerInfo overrideReferrerInfo;
+  bool shouldWaitForOnStartRequestSent;
 };
 
 } // namespace ipc